src/redux/categoriesSlice.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+import categoryReducer from './categories/categories';
+
+const store = configureStore({
+  reducer: {
+    books: booksReducer,
+    categories: categoryReducer,
+  },
+});
+
+export default store;
```

Comment:
- [`OPTIONAL` ] Great job creating your store. Everything looks good so far but for the sake of conversion I would like to suggest that you change the name of this file from `categoriesSlice` to `store`.